docs/en/sql-reference/statements/watch.md

```diff
@@ -3,11 +3,15 @@ slug: /en/sql-reference/statements/watch
 sidebar_position: 53
 sidebar_label: WATCH
 ---
+
 import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+import CloudNotSupportedBadge from '@theme/badges/CloudNotSupportedBadge';
+
 
 # WATCH Statement
 
 <ExperimentalBadge/>
+<CloudNotSupportedBadge/>
```

Comment:
Remove entirely.

Reply:
We support this experimental feature in cloud?